static/styles/subscriptions.css

```diff
@@ -780,7 +782,7 @@ h4.user_group_setting_subsection_title {
             white-space: nowrap;
             max-width: 260px;
 
-            .sub-stream-name {
+            .group-name {
```

Comment:
Removing this from `sub-stream-name` seems like it might impact the stream settings UI; did you mean to append?

Reply:
Ohh sorry that was a mistake. I wanted to append `,.group-name` there. I've fixed and pushed the updated commits.